include/RAJA/policy/cuda/kernel/CudaKernel.hpp

```diff
@@ -203,6 +203,41 @@ __launch_bounds__(BlockSize, 1) __global__
   Exec::exec(private_data, true);
 }
 
+
+/*!
+ * Helper class that handles getting the correct global function for
+ * CudaKernel policies.
+ */
+template<size_t BlockSize, typename Data, typename executor_t, bool fixed = (BlockSize > 0)>
+struct CudaKernelLauncherGetter;
+
+/*!
+ * Helper class specialization for unknown number of threads.
```

Comment:
Please add a comment describing the meaning of 'true' and 'false' in these methods. Thanks.